client/my-sites/comments/comment-list/index.jsx

```diff
@@ -379,11 +379,14 @@ export class CommentList extends Component {
 
 		return (
 			<div className="comment-list">
-				{ isJetpack &&
-					<QueryJetpackSiteCommentsTree page={ page } siteId={ siteId } status={ status } />
-				}
-				{ ! isJetpack &&
-					<QuerySiteCommentsTree siteId={ siteId } status={ status } />
+				{ isJetpack
+					? <QuerySiteCommentsList
+						number={ 100 }
+						offset={ ( page - 1 ) * COMMENTS_PER_PAGE }
+						siteId={ siteId }
+						status={ status }
+					/>
+					: <QuerySiteCommentsTree siteId={ siteId } status={ status } />
```

Comment:
Fun fact: it was in fact separate ifs before, but I replaced it with a ternary because I was expecting your customary "you can use a ternary here" comment. 😆